src/config/index.ts

```diff
@@ -166,6 +166,35 @@ export const RESPONSE_DUE_DATE_FIELD_ID =
  */
 export const GH_USER_TOKEN = process.env.GH_USER_TOKEN || '';
 
+/**
+ * App auth strategy options. For branding purposes, we have one app per org
+ * (getsantry for getsentry, covecod for codecov, &c). Down in getClient we
```

Comment:
nit: "&c" -> "etc"?

Reply:
Bytes are expensive. ;-)

Done in ffb23786d62d39048324d6ac40efd90ad9f368b7.